CRM/Event/Form/ParticipantFeeSelection.php

```diff
@@ -426,4 +426,30 @@ public function getEventID(): int {
     return $this->_eventId;
   }
 
+  /**
+   * Get the price set ID for the event.
+   *
+   * @return int|null
+   *
+   * @api This function will not change in a minor release and is supported for
+   * use outside of core. This annotation / external support for properties
+   * is only given where there is specific test cover.
+   *
+   * @noinspection PhpUnhandledExceptionInspection
+   * @noinspection PhpDocMissingThrowsInspection
+   */
+  public function getPriceSetID(): ?int {
```

Comment:
Should probably use this at 127 as well for consistency.

Reply:
Actually, $priceSetId is unused, we can just remove 127.

Reply:
done!